src/Microsoft.AspNet.IISPlatformHandler/IISWebApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class IISWebApplicationBuilderExtensions
+    {
+        public static IWebApplicationBuilder UseIISPlatformHandler(this IWebApplicationBuilder webApplicationBuilder)
```

Comment:
IWebApplicationBuilder.UseIISPlatformHandler needs to be searchably distinct from IApplicationBuilder.UseIISPlatformHandler. UseIISPlatformHandlerUrl?